packages/step/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@react-spectre/step",
+  "version": "1.0.0-15",
+  "description": "React components for Spectre.css's steps.",
+  "jsnext:main": "src/index.js",
+  "main": "dist/step.js",
+  "browser": "dist/step.umd.js",
+  "module": "dist/step.es.js",
+  "repository":
+    "https://github.com/react-spectre/react-spectre/tree/master/packages/step",
+  "author":
+    "Cesar Contreras <ccdl15c@gmail.com> (http://github.com/ccontreras)",
+  "license": "MIT",
```

Comment:
I may change the authorship here to avoid conflicts with NPM organizations and scoped packages.